student.rb

```diff
@@ -1,18 +1,18 @@
+require_relative 'feedback_module'
+# require_relative 'all_teachers'
+
 class Student
   attr_reader :age, :phase
   attr_accessor :name
+  include Feedback
 
   def initialize(options = {})
     @phase = 1
     @age = options.fetch(:age, 0)
     @name = options.fetch(:name, "")
   end
```

Comment:
This stuff looks pretty similar to the stuff in Teacher and ApprenticeTeacher....